modules/tinymce/src/core/test/ts/browser/DragDropOverridesTest.ts

```diff
@@ -257,6 +257,31 @@ describe('browser.tinymce.core.DragDropOverridesTest', () => {
       });
     });
 
+    it('TINY-9025: Dragging CEF element outside the editor should not cause scrolling', async () => {
+      const editor = hook.editor();
+      editor.setContent(`
+      <p>CEF can get dragged before this one</p>
+      <p class="hidden" style="height: 1500px"></p>
+      <p id="separator" style="height: 100px"></p>
+      <p contenteditable="false" style="height: 200px; background-color: black; color: white">Draggable CEF</p>
+    `);
+      window.scroll({
+        top: window.innerHeight + 2000
+      });
+      const target = UiFinder.findIn(TinyDom.body(editor), 'p:contains("Draggable CEF")').getOrDie();
+      const initialScrollY = window.scrollY;
+      editor.getWin().scroll({
+        top: editor.getWin().innerHeight
+      });
+      Mouse.mouseDown(target);
+      window.dispatchEvent(new MouseEvent('mousemove', {
+        clientY: window.innerHeight,
+      }));
+      await Waiter.pWait(100);
```

Comment:
Can this be a wait for waiter maybe with an interval of 100 if the setInterval of the scroll thing is 100. I always get a bit worried that this might misbehave if the timing is slightly off since the browser on the VM is slow.

Reply:
Right this is a waiter for something to now happen since it compares with the initial. So then I guess it's fine to have just a wait here since we don't know how long we need to vapid for that condition to not be true.